robot_servicio/scripts/send_goals.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import rospy
+from geometry_msgs.msg import Pose, Quaternion
+from tf.transformations import quaternion_from_euler
+
+
+class send_goals():
+    def __init__(self):
+        self.pi = 3.1416
+
+        sec_1_table_1 = Pose()
```

Comment:
Algo que estaria bueno considerar pero no sé cuan bien funciona, sería usar algo así como "base de datos" y el robot puede ir leyend esas poses: https://strands.readthedocs.io/en/latest/strands_navigation/topological_navigation/#creation-of-the-topological-map-from-waypoint-file

Reply:
Esto también se los mencioné en el anteproyecto y creo que sería un muy buen feature: https://strands.readthedocs.io/en/latest/lamor15/wiki/Tutorial-materials-1.html#creating-the-topological-map